code/modules/mob/hear_say.dm

```diff
@@ -61,12 +61,12 @@
 		if(get_preference_value(/datum/client_preference/ghost_ears) == GLOB.PREF_ALL_SPEECH && (can_see(speaker)))
 			message = "<b>[message]</b>"
 
-	if(!can_hear(speaker) || get_sound_volume_multiplier() < 0.2)
+	if(!can_hear() || get_sound_volume_multiplier() < 0.2)
 		if(!language || !(language.flags & INNATE)) // INNATE is the flag for audible-emote-language, so we don't want to show an "x talks but you cannot hear them" message if it's set
 			if(speaker == src)
 				to_chat(src, SPAN_WARNING("You cannot hear yourself speak!"))
 			else if(can_see(speaker))
-				to_chat(src, "<span class='name'>[speaker_name]</span>[alt_name] talks but you cannot hear \him.")
+				to_chat(src, "<span class='name'>[speaker_name]</span>[alt_name] says something you cannot hear.")
```

Comment:
```suggestion
				to_chat(src, "<span class='name'>[speaker_name]</span>[alt_name] says something you cannot hear [speaker.p_them()].")
```

Reply:
but you cannot hear*
also why deviate from upstream bay? https://github.com/Baystation12/Baystation12/blob/dev/code/modules/mob/hear_say.dm#L50

Reply:
Because I do not care about upstream bay.